specification/transport_layer/transport_layer.tex

```diff
@@ -379,7 +386,11 @@ \subsection{Redundant interface support}
 utilization rates of the redundant interfaces and different phasing of their traffic;
 however, that is not an issue for UAVCAN.
 If perfectly simultaneous frame submission is not possible, interfaces with lower numerical index
-should be handled in the first order.
+should be handled in the first order\footnote{It should be kept in mind that redundant interfaces
+are used for increased fault tolerance, not for load sharing reasons.
+Whenever a node is connected to an interface, the likelihood of its failure is increased;
+therefore, backup interfaces should interconnect only mission-critical equipment.
```

Comment:
This is a non-normative comment so we shouldn't use "should" here. Perhaps: This suggests that backup interfaces may only interconnect with mission-critical equipment.

I know there is a counter argument to be made in having asymmetrical redundant busses where it adds complexity because the testing matrix is increased (i.e. you have to prove that mission-critical component X can operate nominally on bus B where node Y,Z, and Q are no longer available). This even makes the wiring harnesses different per bus which is, perhaps, added complexity and does mean the bus capacitance has to be calculated differently. Considering there is very little that is not mission-critical on some drones (i.e. where your ESCs are the only thing on the bus) I wouldn't put too much weight on this footnote.